src/frontend/cart/CartItem.js

```diff
@@ -0,0 +1,56 @@
+import './cart.css';
+import { useCartCtx } from '../context/cartContext';
+import { useWishlistCtx } from '../context/wishlistContext';
+import { useCartAPICtx } from '../context/cartContext';
```

Comment:
You can use barrel exports, so that you wont have to import each file separately